src/routes/v2/auth.js

```diff
@@ -82,23 +72,14 @@ class AuthRouter {
   async verify(req, res) {
     const { email: rawEmail, otp } = req.body
     const email = rawEmail.toLowerCase()
-    const token = await this.authService.verifyOtp({ email, otp })
-    const authTokenExpiry = new Date()
-    // getTime allows this to work across timezones
-    authTokenExpiry.setTime(authTokenExpiry.getTime() + AUTH_TOKEN_EXPIRY_MS)
-    const cookieSettings = {
-      path: "/",
-      expires: authTokenExpiry,
-      httpOnly: true,
-      sameSite: true,
-      secure: isSecure(),
-    }
-    res.cookie(COOKIE_NAME, token, cookieSettings)
+    const userInfo = await this.authService.verifyOtp({ email, otp })
+    Object.assign(req.session, { userInfo })
     return res.sendStatus(200)
   }
 
   async logout(req, res) {
     this.clearIsomerCookies(res)
+    req.session.destroy()
```

Comment:
destroy is a hard delete!

For the login on alternate source, as discussed offline, this is something which would be good to have, but we're opting to file an issue about this for now as this is a minor case

Reply:
Sounds ok to me but could we just check w Kathleen whether we should log or preserve this? (Only asking cos of the qns we got wrt security)

Reply:
@seaerchin hmm even if there is a requirement to preserve this, do you think it would be better to preserve this at the cloudwatch logs level rather than our db? 
Only saying this as preserving this would lead to an extremely long table, not sure if this might be ideal for our use case!

Reply:
err i think either work? the main concern w/ storing cloudwatch is retrievability - we'd need to store logs well enough, together with their meta properties (eg: which user is this) in order for it to be useful. 

wrt long table - cos `destroy` is a hard delete, idt the table would be extremely long (once logout, the row is gone) but correct me if i'm wrong

Reply:
yupyup, but if we soft delete it will become an extremely long table mah! 

My pov is as long as we have the unique identifier for a user logged as their meta, i think is ok (@alexanderleegs please correct me if we dont already do this)